apps/admin-gui/src/app/shared/components/add-owner-dialog/add-owner-dialog.component.html

```diff
@@ -0,0 +1,56 @@
+<div class="user-theme">
+  <h1 mat-dialog-title>{{"DIALOGS.ADD_OWNER.TITLE" | translate}}</h1>
+  <mat-spinner class="ml-auto mr-auto" *ngIf="loading"></mat-spinner>
+  <div *ngIf="!loading">
+    <div mat-dialog-content class="dialog-container">
+      <mat-form-field>
+        <input
+          matInput
+          required
+          placeholder="{{'DIALOGS.ADD_OWNER.NAME' | translate}}"
+          [formControl]="nameCtrl">
+        <mat-error>
+          {{'DIALOGS.ADD_OWNER.FILL_NAME' | translate}}
+        </mat-error>
+      </mat-form-field>
+      <mat-form-field>
+        <input
+          matInput
+          required
+          placeholder="{{'DIALOGS.ADD_OWNER.CONTACT' | translate}}"
+          [formControl]="contactCtrl">
+        <mat-error *ngIf="contactCtrl.errors.required">
+          {{'DIALOGS.ADD_OWNER.FILL_CONTACT' | translate}}
+        </mat-error>
```

Comment:
If I put the correct email into contact form field, the dialog partly freezes and app throws this error. You can try `contactCtrl.hasError('required')`.

![required_err](https://user-images.githubusercontent.com/70371638/111387507-1b934580-86ae-11eb-84b8-f8d82de95b3d.png)